src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -171,6 +171,24 @@ Engine::Engine() :
     deviceFullName(getDeviceFullName()),
     specialSetup(new CPUSpecialSetup) {
     set_device_name("CPU");
+    auto proc_type_table = get_proc_type_table();
```

Comment:
As I mentioned on the sync that makes a perfect sense to extract task executor creation logic into separate function (ideally places in threading interface, not the plugin). Function signature might take core type and std::fucntion. Then from plugin perspective it might be one-line usage (or smt similar):
```
execute_task(PCORE, []{dnnl::impl::cpu::x64::cpu()});
```

Reply:
The function was extracted and was placed in class ExecutorManager.